features/karaf/openhab-core/src/main/feature/feature.xml

```diff
@@ -543,6 +543,7 @@
     <!-- It is temperarily disabled due to https://github.com/openhab/openhab-core/issues/422 -->
     <!-- <bundle start-level="30">mvn:org.openhab.core.bundles/org.openhab.ui.start/${project.version}</bundle> -->
```

Comment:
Issue #422 has been fixed / closed some time ago :)

Reply:
I guess that line needs to be removed anyway, because there is no `org.openhab.ui.start` in this core repo.

Reply:
More details: https://github.com/openhab/openhab-core/issues/422#issuecomment-435622005